alertwildfire_live_recording.py

```diff
@@ -68,23 +68,30 @@ def capture_and_record(googleServices, outputDir, camera_name):
     Returns:
         imgPath: local path to downloaded object
     """
-    imgPath = alertwildfire_API.request_current_image(outputDir, camera_name)
-    cloud_file_path = camera_name + '/' + pathlib.PurePath(imgPath).name
-    goog_helper.uploadBucketObject(googleServices["storage"],"fuego-firecam-a",cloud_file_path, imgPath)
+    success = False
+    while not success:
+        pull1 = alertwildfire_API.get_individual_camera_info(camera_name)
+        imgPath = alertwildfire_API.request_current_image(outputDir, camera_name) 
+        pull2 = alertwildfire_API.get_individual_camera_info(camera_name)
+        if pull1['position'] == pull1['position']:
+            success = True
+
+    image_base_name = pathlib.PurePath(imgPath).name
+    image_name_with_metadata = image_base_name[:-23]+'p'+str(pull1['position']['pan'])+'_t'+str(pull1['position']['tilt'])+'_z'+str(pull1['position']['zoom'])+  '__'+image_base_name[-23:]
```

Comment:
We should clean this up in the future, but let's push this in as is to get started.